docs/developer-docs/integrations/sns/launching/launch-summary.md

```diff
@@ -0,0 +1,332 @@
+| Initial developer neurons                         | Pre-decentralization swap mode        | Dapp developer principal  |
+| Treasury account on the SNS Ledger                | Pre-decentralization swap mode        | Newly-created SNS canisters on the SNS subnet   |
+| Swap account on the SNS Ledger                    | Pre-decentralization swap mode        | Newly-created SNS canisters on the SNS subnet   |
+
+
+- #### Step 6: Dapp developers submit an SNS proposal to handover control of their dapp to the SNS.
+  Before the decentralization swap, the developers hand over the control of the dapp to the SNS.
+
+  This includes adding the SNS root canister as the controller of the dapp and removing
+  yourself (and possibly other developers) from the list of
+  controllers. Also, this includes "registering" the dapp with the SNS so that SNS
+  root is aware that it controls these canisters.
+
+  The registration is done by **SNS proposal**.
+  
+  As there are already initial neurons (from Step 3), this,
+  as well as potential upgrades to the dapp, can be realized by SNS proposals and by the
+  majority of initial neurons voting in favor of them.
+  The initial neurons cannot do other things, such as changing
+  the SNS  parameters, as the SNS governance canister is still in
+  pre-decentralization-swap mode.
+
+  :::info
+  Note that this is an *SNS* proposal, not an NNS proposal, even though many users would vote on it within [the NNS frontend dapp](nns.ic0.app).
+  :::
+
+  If successful, at the end of stage, we have:
+
+
+|  Decentralization swap state                      | PENDING                                    |  
+|-------------------                                | ----------------                        |
+
+
+|  Objects in an app subnet                                      | State                    |  Controlled by |
+|-------------------                                | ----------------                      | ----------------|
+| A dapp                                            | Operational                           | Dapp developer principal | 
+
+|  Objects in the SNS subnet                                       | State                  |  Controlled by |
+|-------------------                                | ----------------                      | ----------------|
+| ~~A principal that can deploy to SNS subnet~~     | Revoked because it is 1-time use      | NA                        |
+| Newly-created SNS canisters on the SNS subnet     | Pre-decentralization swap mode        | Initial developer neurons |  
+| Initial developer neurons                         | Pre-decentralization swap mode        | Dapp developer principal  |
+| Treasury account on the SNS Ledger                | Pre-decentralization swap mode        | Newly-created SNS canisters on the SNS subnet   |
+| Swap account on the SNS Ledger                    | Pre-decentralization swap mode        | Newly-created SNS canisters on the SNS subnet   |
```

Comment:
would it help to add here "pending proposal" to the state? I was first confused wrt what is now different compared to the last step.

Reply:
yes, added them